processing/src/main/java/org/apache/druid/segment/column/ComplexColumn.java

```diff
@@ -19,45 +19,34 @@
 
 package org.apache.druid.segment.column;
 
+import org.apache.druid.guice.annotations.ExtensionPoint;
 import org.apache.druid.query.monomorphicprocessing.RuntimeShapeInspector;
 import org.apache.druid.segment.ColumnValueSelector;
 import org.apache.druid.segment.ObjectColumnSelector;
-import org.apache.druid.segment.data.GenericIndexed;
 import org.apache.druid.segment.data.ReadableOffset;
 
 import javax.annotation.Nullable;
 
 /**
-*/
-public class ComplexColumn implements BaseColumn
+ * This interace represents a complex column and can be implemented by druid extension writer of a custom column
```

Comment:
interace -> interface?

Reply:
that was intentional to see if you were actually reading the doc update.